.github/workflows/ci.yml

```diff
@@ -24,13 +24,8 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
     
-    - name: Add GOPATH/bin to PATH
-      # temporary fix
-      # see https://github.com/actions/setup-go/issues/14
-      run: |
-        set -euo pipefail
-        echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
-        echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+    - name: Add GOPATH
```

Comment:
This name seems wrong to me. 

`GOPATH` is already set but we're adding its bin to `PATH`.

Reply:
Yeah the name could be better like Add GOPATH/bin to PATH.

Reply:
sent #493, let me know what you think.